Client/Frontend/Browser/Tabs/InactiveTabs/InactiveTabsFooterView.swift

```diff
@@ -22,25 +20,21 @@ class InactiveTabsFooterView: UICollectionReusableView, ReusableCell, ThemeAppli
     // MARK: - Properties
     var buttonClosure: (() -> Void)?
 
-    let containerView = UIView()
-    var shouldLeftAlignTitle = false
-
     // MARK: - UI Elements
-    private lazy var roundedButton: UIButton = {
-        let button = UIButton()
+    private lazy var roundedButton: UIButton = .build { button in
         button.titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
             withTextStyle: .body,
             size: UX.buttonFontSize)
-        button.setImage(UIImage(systemName: "trash"), for: .normal)
+        button.setImage(UIImage(systemName: ImageIdentifiers.trash), for: .normal)
```

Comment:
Should we maybe use `StandardImageIdentifiers.Large.delete` instead of a system image?